CODE_OF_CONDUCT.md

```
@@ -1,23 +1,128 @@
-This Code of Conduct applies in all project spaces and communication channels related to the project, both public and private.
+This Code of Conduct applies within all community spaces, and also applies when
+an individual is officially representing the community in public spaces.
+Examples of representing our community include using an official e-mail address,
+posting via an official social media account, or acting as an appointed
+representative at an online or offline event.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported to the community leaders responsible for enforcement at
+ricciutipaolo@gmail.com.
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)